packages/advi/advi.2.0.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "Active DVI Dune package!"
+description: """
+
+Active-DVI is a DVI previewer and presenter written in OCaml with some eye
+candy effects for presentation, support for interactive demonstrations, and
+embedding of arbitrary applications within the presentation.
+
+Active-DVI is well suited for use in combination with WhizzyTeX
+(http://cristal.inria.fr/whizzytex/)
+"""
+maintainer: ["Didier Rémy <Didier.Remy@inria.fr>"]
+authors: [
+  "Alexandre Miquel"
+  "Jun Furuse"
+  "Didier Rémy"
+  "Pierre Weis"
+  "Xavier Leroy"
+  "Roberto Di Cosmo"
+  "Didier Le Botlan"
+  "Alan Schmitt"
+]
+license: "LGPL-2.1-or-later"
+homepage: "http://advi.inria.fr"
+dev-repo: "git+https://github.com:diremy/advi.git"
+doc: "http://advi.inria.fr"
+bug-reports: "Didier.Remy@inria.fr"
+depends: [
+  "ocaml-base-compiler" {>= "4.11.1"}
+  "dune" {>= "2.5"}
+  "graphics" {>= "5.1.1"}
+  "camlimages" {>= "5.0.4"}
+  "conf-texlive"
+  "conf-which"
+]
+build: [
+    ["dune" "subst"] {pinned}
```

Comment:
```suggestion
    ["dune" "subst"] {dev}
```
Sorry, I had missed this

Reply:
You don't need to re-tag or re-release for this after you make the change upstream.